.github/workflows/int-load-test.yml

```diff
@@ -0,0 +1,34 @@
+name: k6 Load Test (INT)
+
+on: [workflow_dispatch]
+
+jobs:
+  k6:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Testing GraphQL - Components
+        if: always()
+        uses: grafana/k6-action@v0.2.0
+        with:
+          filename: k6/graphql/int/components.js
+          cloud: true
+          token: ${{ secrets.K6_CLOUD_API_TOKEN }}
+
+      - name: Testing GraphQL - Observations
+        if: always()
+        uses: grafana/k6-action@v0.2.0
+        with:
+          filename: k6/graphql/int/observations.js
+          cloud: true
+          token: ${{ secrets.K6_CLOUD_API_TOKEN }}
+
+      - name: Testing HAR - Edition
+        if: always()
+        uses: grafana/k6-action@v0.2.0
+        with:
+          filename: k6/har/int/edition.js
+          cloud: true
+          token: ${{ secrets.K6_CLOUD_API_TOKEN }}
```

Comment:
Could we find a way to have all three env in the same file ? It seems it will be a pain to maintain otherwise.

Reply:
Yes, that should be possible, we could have some dropdown then when running the action manually to select the env. I'll take a look 👀

Reply:
See https://github.com/visualize-admin/visualization-tool/pull/1076/commits/b27c8a0659d43338bb73e3b1d2d65836b01db476 :)